_drafts/2019-02-27-face-anti-spoofing.md

```diff
@@ -0,0 +1,149 @@
+
+## Ne hakkında konuşacağız
+
+Biyometrik tanımlama, en eski kişi doğrulama tekniklerinden biridir. Parolalar ve anahtarlar casusluk yoluyla elde edilebilir, çalınabilir, unutulabilir veya taklit edilebilir.  Ancak kişinin kendisinin benzersiz özelliklerini taklit etmek ve kaybetmek çok daha zordur. Bu özellikler; parmak izleri, ses, retinanın damarlarının şekli ve daha fazlası olabilir.
+
+Ancak tahmin ettiğiniz üzere biyometrik sistemleri de kandırmaya çalışanlar var. İşte bu makalenin konusu da: Saldırganlar başka bir kişiyi taklit ederek yüz tanıma sistemlerinden nasıl kaçınmaya çalışıyor ve bu nasıl tespit edilebilir!
+
+**Bu makaleyi okuduktan sonra neler hakkında bilgi sahibi olacaksınız:**
+
+ - Yüz tanıma (face recognition) ve yüz sahteciliği (face spoofing) arasındaki ilişki
+ - Yüz sahteciliği önleme (face anti-spoofing) alanında kullanılan temel terimler ve metrikler  
+ - Yüz sahteciliği saldırı türleri  
+ - Yüz sahteciliğini önleme yöntemlerinin uygulama alanları  
+ - Seçilen akademik yüz sahteciliğini önleme makalelerinin kilit noktaları
+ - Yüz sahteciliği veri kümelerindeki geçerli mevcut durum
+
+## Yüz tanıma teknolojisinden yüz sahteciliğine
+Günümüzün yüz tanıma sistemleri muazzam bir doğruluk göstermektedir. Büyük veri setlerinin ve karmaşık mimarilerin ortaya çıkmasıyla, **0.000001'e (milyonda bir hata!) kadar yüz tanıma doğruluğu** elde etmek mümkün hale geldi ve artık mobil platformlara aktarma için uygunlar. Bu yüz tanıma sistemlerinin günlük kullanımının yaygınlaşmasına tek engel; onların zayıf taraflarıydı.
+
+Teknik gerçekliğimizde başka bir kişiyi taklit etmek için en sık  kullanılan yöntem maskelerdir. Örneğin, aşağıdaki resimde olduğu gibi siyahi bir maske takarak bir bankayı soyun:
+
+![](https://miro.medium.com/max/1170/0*-AA1QJl_qiwtADqu)
+Yüz tanıma sistemlerini kandırmanın en iyi yolu kişinin kendi yüzü yerine başka birinin yüzünü sunmasıdır. Maskeler; yazıcıdan çıktısı alınan başkasına ait yüzün fotoğrafından, ısıtmalı çok karmaşık üç boyutlu maskelere kadar tamamen farklı kalitede olabilir, ayrı bir sayfa veya ekran şeklinde de sunulabilir, kişinin başına da takılabilir.
+
+Samsung'daki **Iris tarayıcısı** ve iPhone X'teki  **Face ID** sistemini aldatma girişimi ile birlikte, yüz sahteciliği konusuna olan ilgi de artmaya başladı. Aşağıdaki şekilde, Vietnam güvenlik şirketi olan Bkav'ın, **Face ID'yi aşmak için kullandığı maske**yi görüyorsunuz. Maske, silikon burun, 3 boyutlu yazıcıdan çıkarılmış parçalar ve 2 boyutlu fotoğraflardan oluşuyor. Detaylı bilgiye [bu videodan da](https://www.youtube.com/watch?v=kSwzTqM3t_0) ulaşabilirsiniz.
+
+![](https://miro.medium.com/max/692/0*AdUio17zq30vgfvz)
+Bu tür güvenlik açıklarının varlığı, bir saldırganın nüfuz etmesinin önemli kayıplara yol açabilecği bankacılık veya şehir güvenlik sistemlerinde otomatik yüz doğrulama sistemlerinin kullanımını tehlikeye sokmaktadır.
```

Comment:
yazının resmin yanından değil de altından devam etmesi için resimden sonra bir satır boşluk bırakmak gerekli